modules/platforms/cpp/core-test/src/teamcity_boost.cpp

```diff
@@ -80,6 +80,7 @@ class TeamcityBoostLogFormatter : public boost::unit_test::unit_test_log_formatt
     virtual void log_start(std::ostream&, boost::unit_test::counter_t);
     virtual void log_finish(std::ostream&);
     virtual void log_build_info(std::ostream&);
+    virtual void log_build_info(std::ostream&, bool);
```

Comment:
Why was it added?

Reply:
> Why was it added?

I used boost 1.70.0, the signature of function virtual void log_build_info(std::ostream&)
(see here: https://github.com/boostorg/test/blob/boost-1.70.0/include/boost/test/unit_test_log_formatter.hpp#L144) was changed comparing to version 1.69.0:
https://github.com/boostorg/test/blob/boost-1.69.0/include/boost/test/unit_test_log_formatter.hpp#L143

So the compilation error had place and it was needed to fix it.